antarest/core/exceptions.py

```diff
@@ -8,6 +8,49 @@ class ShouldNotHappenException(Exception):
     pass
 
 
+class STStorageFieldsNotFoundError(HTTPException):
+    """Fields of the short-term storage are not found"""
+
+    def __init__(self, storage_id: str) -> None:
+        detail = f" fields of storage '{storage_id}' not found"
+        super().__init__(HTTPStatus.NOT_FOUND, detail)
+
+    def __str__(self) -> str:
+        return self.detail
```

Comment:
any particular reason why you keep the return here ? the `__init__` method doesn't need to return anything